integration-tests/utils/integration-utils.js

```diff
@@ -159,8 +159,13 @@ const select = async (page, labelContains, text) => {
   return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
 }
 
-const waitForNoSpinners = page => {
-  return page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
+const waitUntilDomContentLoaded = page => {
+  return page.waitForSelector('div:not(empty):not([id="root"])') // Used to prevent checking loadingSpinner while page is blank
```

Comment:
This CSS selector should be a good start for most pages but probably not sufficient for edge cases.